4.6.2/Dockerfile

```diff
@@ -1,5 +1,14 @@
-FROM microsoft/iis
+FROM microsoft/dotnet-framework:4.6.2
 
-RUN powershell -Command Add-WindowsFeature NET-Framework-45-ASPNET; \
-    powershell -Command Add-WindowsFeature Web-Asp-Net45; \
-    powershell -Command Remove-Item -Recurse C:\inetpub\wwwroot\*
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
```

Comment:
Perhaps a convention should be used for PowerShell / non-PowerShell images. Similar to many images having a `-alpine` variant, allowing users to either use alpine as a base, or debian.

If PowerShell will be the default, having a `-cmdexe` (or whatever) as an alternative would help for that.

A brief description of the differences (e.g. escaping, if different) in the readme would be good for those